### DISCUSSION TITLE
Update Zooz Firmware Versions to Latest

### DISCUSSION THREAD

**Discussion by DeepestWaters** [zwave-js/zwave-js#7462] [General] [answered]
@AlCalzone thanks for managing the [Zooz configs](https://github.com/zwave-js/node-zwave-js/tree/master/packages/config/config/devices/0x027a).

[More recent firmwares](https://www.support.getzooz.com/kb/article/1158-zooz-ota-firmware-files/) are available for the ZEN72 800LR, ZEN77 800LR, and probably other devices.

Can you/we update the config files accordingly?

Or more ambitiously, add a feature for the updater to periodically check the firmware page and update the config files (which adds risk ofc)?

FYSA Zooz's change logs are out of date but the firmware page has the latest. Their support just kindly updated the [ZEN 72 change log](https://www.support.getzooz.com/kb/article/849-zen72-dimmer-change-log/) and said ZEN 77 is next. 

I code a bit; let me know if I can help.

**Comment by AlCalzone** [comment, marked as answer]
Zooz provide the updates themselves, e.g. here: https://github.com/zwave-js/firmware-updates/pull/144
You could ask them to post the latest round of updates.

**Reply by DeepestWaters** [reply]
Will do; thanks!
